ehrql/debugger.py

```diff
@@ -161,3 +194,11 @@ def related_event_columns_to_records(columns):
                     column[patient_id][row_id], convert_null=True
                 )
             yield record
+
+
+def is_ehrql_object(element):
+    isDataset = isinstance(element, Dataset)
+    isSeries = isinstance(element, BaseSeries)
+    isDateDifference = isinstance(element, DateDifference)
```

Comment:
`isinstance` can take a tuple, so this can be one `isinstance(element, (Dataset, BaseSeries, DateDifference))`

Reply:
Thanks. I've changed it to `isinstance(element, BaseSeries | DateDifference | Dataset)`.